core/templates/components/forms/schema-based-editors/schema-based-list-editor.component.html

```diff
@@ -48,12 +48,11 @@
       Please ensure that the list has no duplicates.
     </div>
 
-    <!-- The margin-left is added here to left-align the button with the previous list items. -->
     <div class="oppia-new-tab-button-container">
-      <button [style.opacity]="(!isOneLineInput || isAddItemButtonPresent || (localValue && localValue.length && localValue[localValue.length - 1].length)) ? 1 : 0"
+      <button
+              [ngClass]="(!isOneLineInput || isAddItemButtonPresent || (localValue && localValue.length && localValue[localValue.length - 1].length)) ? 'btn btn-secondary btn-sm ml-0 e2e-test-add-list-entry' : 'd-none'"
```

Comment:
It will make no change because if the condition in the [ngClass] is false, then the display will be none, so it won't make any difference. Although, please let me know if I should put it inside 'class'

Reply:
Will be better to have it inside `class`!